lms/djangoapps/instructor/views/api.py

```diff
@@ -3032,35 +3032,24 @@ def generate_certificate_exceptions(request, course_id, generate_for=None):
     """
     course_key = CourseKey.from_string(course_id)
 
-    try:
-        certificate_white_list = json.loads(request.body)
-    except ValueError:
-        return JsonResponse({
-            'success': False,
-            'message': _('Invalid Json data, Please refresh the page and then try again.')
-        }, status=400)
-
-    users = [exception.get('user_id', False) for exception in certificate_white_list]
-
     if generate_for == 'all':
```

Comment:
csv is already loaded and also the students get whitelisted too using upload csv button but we are also storing this in backbone collection and send the same data to server for old logic